### ISSUE TITLE
is the loss normal after global step 15000+?

### ISSUE THREAD

**Issue by pbdahzou** [EricZgw/PyramidBox#3]
after global step 15000+, the loss is still very large, is it normal? 

INFO:tensorflow:global step 15489: loss = 24.8054 (0.23 sec/step)
INFO:tensorflow:global step 15499: loss = 21.6636 (0.24 sec/step)
INFO:tensorflow:global step 15509: loss = 26.9590 (0.24 sec/step)
INFO:tensorflow:global step 15519: loss = 26.0390 (0.24 sec/step)
INFO:tensorflow:global step 15529: loss = 23.1402 (0.26 sec/step)

**Comment by EricZgw** [comment]
it's normal,keep training.
